### PR TITLE
[Windows] Add installer projects to VS on Windows-2022

### PR DESCRIPTION
**Pull request by V-Zabayrachny**
# Description
This PR adds  extension "Installer projects" to Visual Studio 2022 on Windows-2022

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by mikhailkoliada — DISMISSED** [review]

File: images/win/scripts/ImageHelpers/InstallHelpers.ps1 [hunk 1]
```diff
@@ -270,10 +271,15 @@ function Install-VsixExtension
     $vsEdition = (Get-ToolsetContent).visualStudio.edition
     try
     {
+        if (Test-IsWin22)
```
Comment (al-cheb) [inline images/win/scripts/ImageHelpers/InstallHelpers.ps1:274]: Why do we need a condition if we have InstallPath param?

Comment (V-Zabayrachny) [reply images/win/scripts/ImageHelpers/InstallHelpers.ps1:274]: In Windows 22 , the location `VSIXInstaller.exe` differs from previous versions of windows, in this regard, a condition was added to override the `InstallPath` variable.

Comment (al-cheb) [reply images/win/scripts/ImageHelpers/InstallHelpers.ps1:274]: That's why param InstallPath is useless if we always rewrite it.

Comment (V-Zabayrachny) [reply images/win/scripts/ImageHelpers/InstallHelpers.ps1:274]: Do you think it should just be moved to a variable?

Comment (al-cheb) [reply images/win/scripts/ImageHelpers/InstallHelpers.ps1:274]: Yep, it should be a variable.

Comment (V-Zabayrachny) [reply images/win/scripts/ImageHelpers/InstallHelpers.ps1:275]: I fixed it. Thanks.

**Review by dmitry-shibanov — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

**Review by al-cheb — APPROVED** [review]

**Pull request merged**